cmp/filter/FilterChooserModel.js

```diff
@@ -125,8 +130,8 @@ export class FilterChooserModel {
             }
         }
 
-        this.setValue(initialValue);
-        this.setFavorites(initialFavorites);
+        this.setValue(isFunction(initialValue) ? initialValue() : initialValue);
```

Comment:
nice!  

could add a utility here, to encourage this pattern.  just not sure what to call it.  `evalIfNeeded()', `callOrReturn()`, `evalIfFn()`, `callIfLazy`,  `lazyEval`, ...